Applications/Viqui/vqCore.cxx

```diff
@@ -3387,6 +3396,43 @@ void vqCore::generateReport(QString path, bool generateVideo)
   this->exportKml(path);
 }
 
+//-----------------------------------------------------------------------------
+void vqCore::exportIqr()
+{
+  this->updateIqrModel();
+
+  QString path =
+    vgFileDialog::getSaveFileName(
+      nullptr, "Save IQR model...", QString(), "IQR File (*.iqr);;");
+
+  if (path.isEmpty())
+    {
+    return;
+    }
+
+  QFile file(path);
+  if (!file.open(QIODevice::WriteOnly |
+                 QIODevice::Truncate))
+    {
+    qDebug() << file.errorString();
```

Comment:
Ick. It is... unfortunate that there are other places employing this pattern. If this is inspected at all (and I agree it probably should be), please instead follow the much better examples in e.g. `vqVqrExporter.cxx` or `vqQueryDialog.cxx` and include it as part of the message box text.